macOS/DuckDuckGo/TabBar/View/TabBarViewItem.swift

```diff
@@ -372,6 +375,9 @@ final class TabBarItemCellView: NSView {
             permissionButton.frame = NSRect(x: maxX - 20, y: bounds.midY - 12, width: 24, height: 24)
         }
 
+        if visualStyle.tabStyleProvider.extraTabSpacing {
```

Comment:
Couldn’t `extraTabSpacing` just be a `CGFloat` and not a `Bool`?

Reply:
You are 💯 right